cycli/lexer.py

```diff
@@ -21,10 +21,10 @@ class CypherLexer(RegexLexer):
              r'|RAND|RANGE|REDUCE|REL|RELATIONSHIP|RELATIONSHIPS|REPLACE|RIGHT|ROUND|RTRIM'
              r'|SHORTESTPATH|SIGN|SIN|SPLIT|SQRT|STARTNODE|STDEV|STDEVP|STR|SUBSTRING'
              r'|SUM|TAIL|TAN|TIMESTAMP|TOFLOAT|TOINT|TRIM|TYPE|UPPER|ALL'
-             r'|AND|ANY|HAS|IN|NONE|NOT|OR|SINGLE|XOR|AS'
+             r'|AND|ANY|HAS|IN|NONE|NOT|OR|SINGLE|XOR|AS|LIKE|ILIKE'
              r'|ASC|ASCENDING|ASSERT|BY|CASE|COMMIT|CONSTRAINT|CREATE|CYPHER'
              r'|DELETE|DESC|DESCENDING|DISTINCT|DROP|ELSE|END|EXPLAIN|FALSE|FIELDTERMINATOR'
-             r'|FOREACH|FROM|WITH HEADERS|IN|INDEX|IS|LIMIT|LOAD|MATCH|MERGE'
+             r'|FOREACH|FROM|WITH HEADERS|INDEX|IS|LIMIT|LOAD|MATCH|MERGE'
```

Comment:
I think we should just alphabetize the whole thing so it's easier to add / find keywords, because LIKE is already there (but it was somewhere else above).